content/blog/similar-document-search-using-apache-spark-with-tf-idf.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Similar Document Search using Apache Spark with TF-IDF
+date: 2021-06-15T06:09:32.919Z
+author: Prasad Singathi, Maikel Pereira
+authorimage: /img/maikel-pereira.jpg
+tags:
+  - MapR
```

Comment:
As tag, in order to tie all MapR related blog posts you submit, may I ask you to add the following tag:

hpe-ezmeral-data-fabric